recipes/anaconda-project/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "anaconda-project" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/Anaconda-Platform/anaconda-project.git
+  git_rev: v{{ version }}
+
+build:
+  number: 0
+  entry_points:
+    - anaconda-project = anaconda_project.cli:main
+
+requirements:
+  build:
+    - anaconda-client
+    - backports_abc >=0.4  # [py2k]
+    - psutil
+    - python
+    - requests
+    - ruamel_yaml
+    - setuptools
+    - singledispatch
+    - tornado
+  run:
+    - anaconda-client
+    - python
+    - requests
+    - ruamel_yaml
+    - tornado
+
+test:
+  commands:
+    - anaconda-project --help
+    - anaconda-project --version
+
+about:
+  home: https://github.com/Anaconda-Platform/anaconda-project
+  license: BSD 3-Clause
+  license_family: BSD
```

Comment:
I already did that on the source, but it will have to be for the next release

https://github.com/Anaconda-Platform/anaconda-project/issues/110